File: autoburn.pc/autoburn/msghandler/RxMsgHandlerTest.cs
using autoburn.Net;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace autoburn.MsgHandler
{
    class RxMsgHandlerTest : RxMsgHandlerBase
    {
        public RxMsgHandlerTest(DeviceNetManager d) : base(d, MSG_TYPE_TEST)
        {
        }
        public override void Process()
        {
            base.Process();
            if (string.IsNullOrEmpty(Msg))
            {
                return;
            }
            D("..just for test");

        }
    }
}

File: autoburn.android/app/src/main/java/com/autoburn/msghandler/RxMsgHandlerTest.java
package com.autoburn.msghandler;

import android.util.Log;

/**
 * Created by chowhsienwu on 2017/5/4.
 */
public class RxMsgHandlerTest extends RxMsgHandlerBase {
    public RxMsgHandlerTest(){
        super(MSG_TYPE_TEST);
    }

    @Override
    protected Void doInBackground(String... params) {
        Msg = params[0];

        Log.i(TAG, " the Msg is " + Msg );
        return null;
    }
}

